Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
```

Comment:
Remove "dont touch..." why would I modify it unless I knew what I was doing. 

Maybe we should instead have a test script in /misc/test_vagrant.sh that runs and provisions the vagrant so that ppl can modify at will and see if they break anything.